sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalogSuite.scala

```diff
@@ -318,4 +319,69 @@ class JDBCTableCatalogSuite extends QueryTest with SharedSparkSession {
       assert(msg.contains("Table not found"))
     }
   }
+
+  test("ALTER TABLE case sensitivity") {
+    withTable("h2.test.alt_table") {
```

Comment:
Although I understand this is the patter in this suite, let's declare a variable for this string and reuse it in this new test case, @huaxingao .